Assets/Scripts/Character/Enemies/ShooterEnemy.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace ShiftingDungeon.Character.Enemies
+{
+    using UnityEngine;
+    using Util;
+    using Weapons;
+
+    public class ShooterEnemy : BasicEnemy
```

Comment:
You shouldn't be extending basic enemy especially since you are just overriding the methods anyway.  Extend Enemy and implement as normal.